v3/src/components/graph/components/graph-inspector.tsx

```diff
@@ -1,33 +1,41 @@
-import React, { useRef, useEffect, useState } from "react"
-import { InspectorButton, InspectorMenu, InspectorPanel } from "../../inspector-panel"
+import React, {useRef, useEffect, useState} from "react"
+import {mstAutorun} from "../../../utilities/mst-autorun"
+import {InspectorButton, InspectorMenu, InspectorPanel} from "../../inspector-panel"
 import ScaleDataIcon from "../../../assets/icons/icon-scaleData.svg"
 import HideShowIcon from "../../../assets/icons/icon-hideShow.svg"
 import ValuesIcon from "../../../assets/icons/icon-values.svg"
 import BarChartIcon from "../../../assets/icons/icon-segmented-bar-chart.svg"
 import StylesIcon from "../../../assets/icons/icon-styles.svg"
 import CameraIcon from "../../../assets/icons/icon-camera.svg"
-import { HideShowMenuList } from "./inspector-panel/hide-show-menu-list"
-import { PointFormatPalette } from "./inspector-panel/point-format-panel"
-import { GraphMeasurePalette } from "./inspector-panel/graph-measure-panel"
+import {HideShowMenuList} from "./inspector-panel/hide-show-menu-list"
+import {PointFormatPalette} from "./inspector-panel/point-format-panel"
+import {GraphMeasurePalette} from "./inspector-panel/graph-measure-panel"
 import t from "../../../utilities/translation/translate"
-import { useDndContext } from "@dnd-kit/core"
-import { ITileInspectorPanelProps } from "../../tiles/tile-base-props"
+import {useDndContext} from "@dnd-kit/core"
+import {ITileInspectorPanelProps} from "../../tiles/tile-base-props"
 import {isGraphContentModel} from "../models/graph-content-model"
 
 
-export const GraphInspector = ({ tile, show }: ITileInspectorPanelProps) => {
+export const GraphInspector = ({tile, show}: ITileInspectorPanelProps) => {
   const graphModel = isGraphContentModel(tile?.content) ? tile?.content : undefined
   const [showPalette, setShowPalette] = useState<string | undefined>(undefined)
+  const [, setPlotType] = useState<string | undefined>(undefined)
   const panelRef = useRef<HTMLDivElement>()
   const panelRect = panelRef.current?.getBoundingClientRect()
   const buttonRef = useRef<HTMLDivElement>()
   const buttonRect = buttonRef.current?.getBoundingClientRect()
   const {active} = useDndContext()
 
-  useEffect(()=>{
+  useEffect(() => {
     !show && setShowPalette(undefined)
   }, [active, show])
 
+  useEffect(function respondToChangeInPlotType() {
+    return mstAutorun(() => {
+      setPlotType(graphModel?.plotType)
+    }, { name: "respondToChangeInPlotType" }, graphModel)
+  }, [graphModel])
```

Comment:
Rather than setting up an `autorun` here to synchronize graph model state to local React state, I think you could just make the `GraphInspector` an `observer` and access `graphModel.plotType` directly.